receiver/azureeventhubreceiver/azureresourcelogs_unmarshaler_test.go

```diff
@@ -75,7 +77,7 @@ var maximumLogRecord = func() plog.LogRecord {
 	return lr
 }()
 
-func TestAsTimestamp(t *testing.T) {
+func Test_AsTimestamp(t *testing.T) {
```

Comment:
These changes just add to the diff, mind doing that as a chore later?
```suggestion
func TestAsTimestamp(t *testing.T) {
```

Reply:
No problem, I was just looking at tests in other receivers. I use the kafka and azureblob receiver as examples a lot.